github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -358,7 +358,7 @@ presubmits:
             - "/usr/local/bin/runner.sh"
             - "/bin/sh"
             - "-c"
-            - "export TARGET=k8s-1.22 && export KUBEVIRT_STORAGE=nfs && export CDI_E2E_SKIP=Destructive && automation/test.sh"
+            - "export TARGET=k8s-1.22 && export KUBEVIRT_DEPLOY_NFS_CSI=true && export KUBEVIRT_STORAGE=nfs && export CDI_E2E_SKIP=Destructive && automation/test.sh"
```

Comment:
One nit regarding readability: You can use `env` to declare environment variables for your job run, thus avoiding to put this all into the `command`.

Example: https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml#L181

Reply:
Nice, we should probably refactor all our jobs according to this, as they all provide env vars on the cmd line